src/bidiMapper/domains/network/NetworkUtils.ts

```diff
@@ -23,12 +23,12 @@ import type Protocol from 'devtools-protocol';
 
 import type {Network} from '../../../protocol/protocol.js';
 
-export function computeResponseHeadersSize(headers: Network.Header[]): number {
-  return headers.reduce((total, header) => {
-    return (
-      total + header.name.length + header.value.value.length + 4 // 4 = ': ' + '\r\n'
-    );
-  }, 0);
+export function computeHeadersSize(headers: Network.Header[]): number {
+  const requestHeaders = headers.reduce((acc, header) => {
+    return `${acc}${header.name}: ${header.value.value}\r\n`;
+  }, '');
+
+  return new TextEncoder().encode(requestHeaders).length;
```

Comment:
Apparently TextEncoder is a reliable way to check the size in bytes of a string.